Makefile

```diff
@@ -51,8 +55,8 @@ endif
 
 all: mold mold-wrapper.so
 
-mold: $(OBJS) $(MIMALLOC_LIB)
```

Comment:
Won't this make it not compile the bundled mimalloc even if it should? Maybe unset $(MIMALLOC_LIB) if using the system one, instead of this change.

Reply:
Hmm probably yes, I'll fix it now